.travis.yml

```diff
@@ -244,6 +244,26 @@ matrix:
       compiler: clang
       env: COMPILER='clang++' BUILD_TYPE='Release' USE_CPP14=1
 
+    #6/ Conan package tools
+    - os: linux
+      language: python
+      python: "3.6"
+      sudo: required
+      services:
+        - docker
+      env:
+         CONAN_GCC_VERSIONS: 6.3
+         CONAN_DOCKER_IMAGE: lasote/conangcc63
+         CONAN_REFERENCE: Catch/1.9.6
+         CONAN_CHANNEL: ci
```

Comment:
Yes, now is possible. CONAN_UPLOAD_ONLY_WHEN_STABLE was added in conan-package-tools to solve it. Good point, I'll change to only upload tags.